lib/redux/actions/async.js

```diff
@@ -140,8 +141,13 @@ export function doLogin(creds, opts) {
         memberships: results[2]
       }));
 
-      // detect if a VCA here and redirect to clinic user select screen
+      // detect if a DSA here and redirect to data storage screen
+      const { targetUsersForUpload } = getState();
+      if (!targetUsersForUpload.length) {
```

Comment:
I'm not a fan of the truthiness/lack of consistency here. Can we use the same check as above, perhaps `_.isEmpty`?